### PR TITLE
set default clj-kondo config with common symbols

### PR DESCRIPTION
**Pull request by Liverm0r**
Dont know wheather users would need to refer ^fallback, so didn't move it to clj-kondo.exports (yet).

This configs will be generated inside .clj-kondo/tensegritics:
image

### CODE REVIEW

**Comment by Liverm0r** [comment]
The Idea is to have this autogenerated for users too (I had to copy it myself before this).

**Comment by dupuchba** [comment]
still relevant ?

**Comment by Liverm0r** [comment]
I will fix a conflict and add some global kondo rules, that I missed in my project

File: resources/clj-kondo.exports/tensegritics/clojuredart/config.edn [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {:linters {:non-arg-vec-return-type-hint {:level :off}
-           :flutter/widget {:level :error}} 
+           :flutter/widget {:level :error}
+           :clojure-lsp/unused-public-var {:exclude #{main}}
+           :unresolved-namespace {:exclude [dart dart:core]}
+           :unresolved-symbol {:exclude [bool Function fallback List Map]}}
```
Comment (Liverm0r) [inline resources/clj-kondo.exports/tensegritics/clojuredart/config.edn:5]: I am not sure about fallback here – will users be using this in their code? if not, I will move it back to /root/.clj-kondo.

Comment (dupuchba) [reply resources/clj-kondo.exports/tensegritics/clojuredart/config.edn:5]: Yes they can use fallback

**Comment by dupuchba** [comment]
@Liverm0r tell me when it's good to merge

**Comment by Liverm0r** [comment]
I think it's ok now. I will make another pr with the support for inherit

**Pull request merged**